test-framework/common/src/main/java/io/quarkus/test/common/DefaultDockerContainerLauncher.java

```diff
@@ -137,31 +131,31 @@ public void start() throws IOException {
         }
         args.add(containerImage);
 
-        Path logFile = PropertyTestUtil.getLogFilePath();
-        Files.deleteIfExists(logFile);
-        Files.createDirectories(logFile.getParent());
-
-        Path containerLogFile = Paths.get("target", "container.log");
-        Files.createDirectories(containerLogFile.getParent());
-        FileOutputStream containerLogOutputStream = new FileOutputStream(containerLogFile.toFile(), true);
+        final Path logFile = PropertyTestUtil.getLogFilePath();
+        try {
+            Files.deleteIfExists(logFile);
+            Files.createDirectories(logFile.getParent());
+        } catch (FileSystemException e) {
+            log.warnf("Log file %s deletion failed, could happen on Windows, we can carry on.", logFile);
```

Comment:
The message can probably be limited to Windows

Reply:
Will amend.

Reply:
so on non windows we make it hard fail_